docs/content/policy-testing.md

```diff
@@ -119,6 +119,69 @@ PASS: 3/4
 FAIL: 1/4
 ```
 
+## Enriched Test Report With Variable Values
+
+Sometimes, e.g. when testing rules with complex output, it can be useful to know more about the circumstances that caused a certain expression to fail a test.
+The `--var-values` flag can be used to enrich the test report with the exact expression that caused a test rule to fail, including the values of any variables or references used in the expression.
```

Comment:
A big reason for why this isn't enabled by default is that we must enable the tracer in order to collect variable/ref values, which has a negative impact on performance and memory footprint. For smaller projects with few tests, this isn't an issue, but for larger projects, the negative impact can be huge.
Should we mention this in the docs?

Reply:
Hmm... just a thought, could we re-run failing tests with this enabled? So it wouldn't matter for the passing tests... but maybe I'm overthinking this. (Or a `--rerun-with-var-values` configurable?)

Reply:
💡 that's a good idea!

If you have a lot of failing tests, or a few that pull in lots of data, it could still be enough of an issue to have that enabled by default, perhaps 🤔 .

There's likely some caveats with non-deterministic built-ins as well that we'd need to deal with.

Given the close proximity to the next release, I think we should consider this a feature request for the July release, though. WDYT?